src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx

```diff
@@ -58,6 +60,10 @@ export const RemoveLiquidityModal = () => {
     ...emptyToken,
     ...lpTokenMetadata
   };
+  const token = useMemo(
+    () => tokensList.find(candidateToken => getTokenSlug(candidateToken) === getTokenSlug(lpToken)) ?? lpToken,
```

Comment:
just use `LIQUIDITY_BAKING_LP_SLUG` and remove `lpToken` & `lpTokenMetadata` variables?